easybuild/easyconfigs/n/nvhpc/nvhpc-2020_207-cuda-10.2.eb

```diff
@@ -0,0 +1,80 @@
+# jg (CSCS)
+easyblock = 'Tarball'
+
+name = 'nvhpc'
+version = '2020_207'
+# this recipe installs cuda, do NOT change this line:
+local_cudaversion = '10.2'
+versionsuffix = '-cuda-%s' % local_cudaversion
+homepage = 'https://developer.nvidia.com/hpc-sdk'
+description = """The NVIDIA HPC Software Development Kit (SDK) includes the
+proven compilers, libraries and software tools essential to maximizing
+developer productivity and the performance and portability of HPC
+applications.
+"""
+toolchain = SYSTEM
+source_urls = ['https://developer.download.nvidia.com/hpc-sdk']
+sources = ['%(name)s_%(version)s_Linux_x86_64_cuda_multi.tar.gz']
+checksums = ['71ef81910ae00d2749a23bc1ca6e4a66']
+postinstallcmds = [
+    'pwd && ls -la && '
+    'export NVHPC_SILENT=true && '
+    'export NVHPC_DEFAULT_CUDA=10.2 && '
```

Comment:
Since you are keeping the `local_cudaversion`, maybe you can use the variable in this line?

Reply:
https://github.com/eth-cscs/production/pull/1852/commits/22c9b89358ba3edd397712eb208ed9cd3e2ffadf